src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2397,6 +2397,18 @@ well as menu structures (for main menu and popup menus).
         desc="Clear terminal scrollback buffer"
         windowMode="main"/>
 
+   <cmd id="previousTerminal"
+        label="Previous Terminal"
+        menuLabel="Previous Terminal"
+        desc="Show previous terminal"
+        windowMode="main"/>
```

Comment:
"windowmode = main" causes the main window to be raised and focused before executing the command -- is that what's wanted here?